app/src/main/java/com/it3048/mobiledeviceproject/MeetingDAO.kt

```diff
@@ -1,11 +0,0 @@
-package com.it3048.mobiledeviceproject
-
-import java.text.DateFormat
-
-data class MeetingDAO(
-    var meetingTitle: String,
-    var meetingDate: String,
-    var meetingDescription: String,
-    var meetingLocLink: String
-) {
-}
```

Comment:
Removed "MeetingDAO.kt" as it is not a true DAO as it is not a true DAO class and is unnecessary for utilizing a room database.

Sources: https://developer.android.com/reference/android/arch/persistence/room/Dao
https://developer.android.com/training/data-storage/room/defining-data